ledger/accountdb.go

```diff
@@ -339,7 +339,7 @@ func prepareNormalizedBalancesV6(bals []encodedBalanceRecordV6, proto config.Con
 				return nil, err
 			}
 			ctype := basics.AssetCreatable
-			if resData.IsEmptyApp() {
+			if resData.IsEmptyAppFields() {
```

Comment:
Could you please change this one to `resData.IsEmptyAppFields()` -> `resData.IsApp()` ?
This was the culprit for mismatching digests..

Reply:
done